package.json

```diff
@@ -1,49 +1,34 @@
 {
-  "name": "@buxorg/bux-console",
-  "version": "0.0.0",
```

Comment:
don't bump this number, as currently we don't have a way to bump it automatically, so we will need to do it manually in the PR's which could be annoying, easy to forget and therefore misleading.